cmd/kubeadm/app/node/csr.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/golang/glog"
```

Comment:
We don't use glog in kubeadm (or at least not yet), the default output format is too noisy.